vite.config.ts

```diff
@@ -20,6 +20,282 @@ export default defineConfig({
+            "sizes": "512x512",
+            "type": "image/png",
+            "purpose": "any"
+          },
+          {
+            "src": "/img/icons/android-chrome-1024x1024.png",
+            "sizes": "1024x1024",
+            "type": "image/png"
+          }
+        ],
+        "dir": "ltr",
+        "lang": "en",
+        "orientation": "natural",
+        "start_url": "/index.html",
+        "display": "standalone",
+        "display_override": [
+          "standalone",
+          "browser"
+        ],
+        "related_applications": [
+          {
+            "platform": "itunes",
+            "url": "https://apps.apple.com/app/adamant-messenger/id1341473829"
+          },
+          {
+            "platform": "play",
+            "url": "https://play.google.com/store/apps/details?id=im.adamant.adamantmessengerpwa&pli=1"
+          }
+        ],
+        "prefer_related_applications": true,
+        "shortcuts": [
+          {
+            "name": "Wallet",
+            "url": "/home",
+            icons: [
+              {
+                "src": "/img/icons/android-chrome-96x96.png",
+                "sizes": "96x96",
+                "type": "image/png"
+              },
+            ]
+          },
+          {
+            "name": "Chats",
+            "url": "/chats",
+            icons: [
+              {
+                "src": "/img/icons/android-chrome-96x96.png",
+                "sizes": "96x96",
+                "type": "image/png"
+              },
+            ]
+          },
+          {
+            "name": "Settings",
+            "url": "/options",
+            icons: [
+              {
+                "src": "/img/icons/android-chrome-96x96.png",
+                "sizes": "96x96",
+                "type": "image/png"
+              },
+            ]
+          }
+        ],
+        "screenshots" : [
+          {
+            "src": "./img/screenshots/pwa/1-nightly.adamant.im_chats(iPhone 6_7_8) (1).png",
+            "sizes": "750x1334",
+            "type": "image/png",
+            "platform": "ios"
+          },
+          {
+            "src": "./img/screenshots/pwa/2-msg.adamant.im_home(iPhone 6_7_8) (3).png",
+            "sizes": "750x1334",
+            "type": "image/png",
+            "platform": "ios"
+          },
+          {
+            "src": "./img/screenshots/pwa/3-msg.adamant.im_home(iPhone 6_7_8) (2).png",
+            "sizes": "750x1334",
+            "type": "image/png",
+            "platform": "ios"
+          },
+          {
+            "src": "./img/screenshots/pwa/4-msg.adamant.im_home(iPhone 6_7_8) (2).png",
+            "sizes": "750x1334",
+            "type": "image/png",
+            "platform": "ios"
+          },
+          {
+            "src": "./img/screenshots/pwa/Four-4-images-adamant.png",
+            "sizes": "3030x1334",
+            "type": "image/png",
+            "platform": "ios"
+          },
+          {
+            "src": "./img/ios/ios1.png",
```

Comment:
```suggestion
            "src": "./img/screenshots/ios/ios1.png",
```

wrong path, same for other image below